src/pages/sources/SourcesListView.tsx

```diff
@@ -289,30 +391,28 @@ const SourcesListView: React.FunctionComponent = () => {
       context: ['status', source.connection.status, 'sources']
     });
     return (
-      <Flex gap={{ default: 'gapSm' }}>
-        <FlexItem>
-          <ContextIcon symbol={ContextIconVariant[source.connection.status]} />
-        </FlexItem>
-        <FlexItem>
-          <div>{statusString}</div>
-          {getTimeDisplayHowLongAgo(scanTime)}
-        </FlexItem>
-      </Flex>
+      <Button variant={ButtonVariant.link} onClick={() => {
+        showConnections(source);
+      }}>
+        <ContextIcon symbol={ContextIconVariant[source.connection.status]} />
+        {' '}{statusString}{' '}
+        {getTimeDisplayHowLongAgo(scanTime)}
+      </Button>
     );
   };
 
   return (
     <PageSection variant="light">
       {renderToolbar()}
-      <Table {...tableProps} aria-label="Example things table">
+      <Table {...tableProps} aria-label="Example things table" variant="compact">
         <Thead>
           <Tr>
             <TableHeaderContentWithBatteries {...tableBatteries}>
               <Th {...getThProps({ columnKey: 'name' })} />
               <Th {...getThProps({ columnKey: 'connection' })} />
               <Th {...getThProps({ columnKey: 'type' })} />
               <Th {...getThProps({ columnKey: 'credentials' })} />
-              <Th {...getThProps({ columnKey: 'unreachableSystems' })} />
+              {/* <Th {...getThProps({ columnKey: 'unreachableSystems' })} /> */}
```

Comment:
are unreacheable systems hidden on purpose? if so, kindly delete this Th tag (and the equivalent Td bellow).